prover/lib/compressor/blob/v1/blob_maker.go

```diff
@@ -302,31 +302,30 @@ func (bm *BlobMaker) Equals(other *BlobMaker) bool {
 }
 
 // DecompressBlob decompresses a blob and returns the header and the blocks as they were compressed.
```

Comment:
I think this could use an updated comment on what this function do in a bit more details;

i.e. `DecompressBlob` will parse the header, get a dict checksum, attempt to find the corresponding dict from the store, etc... landed here when trying to follow through the usage of this method with many returns from somewhere else it was not clear :)

Reply:
Would it be better to bundle all the non-error parameters into a response struct>